src/App/Tracker/Controller/Hooks/Listeners/AbstractListener.php

```diff
@@ -325,4 +323,117 @@ protected function createStatus(Github $gitHub, TrackerProject $project, $issueN
+	 * @param   Logger       $logger         Logger object
+	 * @param   object       $project        Object containing project data
+	 * @param   IssuesTable  $table          Table object
+	 * @param   array        $addCategories  The Categories to add
+	 *
+	 * @return  void
+	 *
+	 * @since   1.0
+	 * @throws  \RuntimeException
+	 */
+	protected function setCategories($hookData, Logger $logger, $project, IssuesTable $table, $addCategories)
+	{
+		// The Github ID if we have a pull or issue so that method can handle both
+		$issueNumber = $this->getIssueNumber($hookData);
+
+		if ($issueNumber === null)
+		{
+			$message = sprintf('Error retrieving issue number for %s/%s', $project->gh_user, $project->gh_project);
+
+			$logger->error($message);
+
+			throw new \RuntimeException($message);
+		}
+
+		$categoryModel            = new CategoryModel($this->getContainer()->get('db'));
+		$category['issue_id']     = $table->id;
+		$category['modified_by']  = $this->getGithubBotName($project);
+		$category['categories']   = $addCategories;
+		$category['issue_number'] = $issueNumber;
+		$category['project_id']   = $project->project_id;
+
+		$categoryModel->updateCategory($category);
+	}
+
+	/**
+	 * Get the files modified by the pull request
+	 *
+	 * @param   object  $hookData  Hook data payload
+	 * @param   Github  $github    Github object
+	 * @param   Logger  $logger    Logger object
+	 * @param   object  $project   Object containing project data
+	 *
+	 * @return  array
+	 *
+	 * @since   1.0
+	 */
+	protected function getChangedFilesByPullRequest($hookData, Github $github, Logger $logger, $project)
+	{
+		// Get the files modified by the pull request
+		try
+		{
+			$files = $github->pulls->getFiles($project->gh_user, $project->gh_project, $hookData->pull_request->number);
+		}
+		catch (InvalidResponseCodeException $e)
+		{
+			$logger->error(
+				sprintf(
+					'Error retrieving modified files for GitHub item %s/%s #%d',
+					$project->gh_user,
+					$project->gh_project,
+					$hookData->pull_request->number
+				),
+				['exception' => $e]
+			);
+
+			$files = [];
+		}
+		catch (\DomainException $e)
+		{
+			$logger->error(
+				sprintf(
+					'Error retrieving modified files for GitHub item %s/%s #%d',
+					$project->gh_user,
+					$project->gh_project,
+					$hookData->pull_request->number
+				),
+				['exception' => $e]
+			);
+
+			$files = [];
+		}
+
+		// Retrun the changed files
+		return $files;
+	}
+
+	/**
+	 * Return the currently configured bot account. Fallback is 'joomla-cms-bot'
+	 *
+	 * @param   object  $project  Object containing project data
+	 *
+	 * @return  string  The currently configured bot account
+	 *
+	 * @since   1.0
+	 *
+	 */
+	protected function getGithubBotName($project)
+	{
+		// Look if we have a bot user configured
+		if ($project->getGh_Editbot_User())
+		{
+			// Retrun the user name
+			return $project->getGh_Editbot_User();
+		}
+
+		// Retun "joomla-cms-bot" as fallback
+		return 'joomla-cms-bot';
```

Comment:
Use `joomla-bot` as the fallback instead.  That one I don't think gets used as much and has access to all the same repos as the rest of the botnet.